api/api/models/media.py

```diff
@@ -8,12 +8,12 @@
 from django.utils.html import format_html
 
 from elasticsearch import Elasticsearch, NotFoundError
+from openverse_attribution.attribution import get_attribution_text
```

Comment:
I know we've gone back and forth on the name, but it ocurred to me this doesn't even need openverse specified in the package name. Nothing about it is actually Openverse specific. It is, however, CC-specific! `cc_attribution` would be a good name for it.

Just a nit though, don't bother spending time on this, we can bikeshed the name if we ever get around to publishing it.